tests/load/sources/sql_database/sql_source.py

```diff
@@ -0,0 +1,373 @@
+            Column(
+                "created_at",
+                DateTime(timezone=True),
+                nullable=False,
+                server_default=func.now(),
+            ),
+            Column("name", Text(), nullable=False),
+            Column("active", Boolean(), nullable=False, server_default=text("true")),
+            Column(
+                "updated_at",
+                DateTime(timezone=True),
+                nullable=False,
+                server_default=func.now(),
+            ),
+        )
+        Table(
+            "chat_message",
+            self.metadata,
+            Column("id", Integer(), primary_key=True, autoincrement=True),
+            Column(
+                "created_at",
+                DateTime(timezone=True),
+                nullable=False,
+                server_default=func.now(),
+            ),
+            Column("content", Text(), nullable=False),
+            Column(
+                "user_id",
+                Integer(),
+                ForeignKey("app_user.id"),
+                nullable=False,
+                index=True,
+            ),
+            Column(
+                "channel_id",
+                Integer(),
+                ForeignKey("chat_channel.id"),
+                nullable=False,
+                index=True,
+            ),
+            Column(
+                "updated_at",
+                DateTime(timezone=True),
+                nullable=False,
+                server_default=func.now(),
+            ),
+        )
+        Table(
+            "has_composite_key",
+            self.metadata,
+            Column("a", Integer(), primary_key=True),
+            Column("b", Integer(), primary_key=True),
+            Column("c", Integer(), primary_key=True),
+        )
+
+        def _make_precision_table(table_name: str, nullable: bool) -> None:
+            Table(
+                table_name,
+                self.metadata,
+                Column("int_col", Integer(), nullable=nullable),
+                Column("bigint_col", BigInteger(), nullable=nullable),
+                Column("smallint_col", SmallInteger(), nullable=nullable),
+                Column("numeric_col", Numeric(precision=10, scale=2), nullable=nullable),
+                Column("numeric_default_col", Numeric(), nullable=nullable),
+                Column("string_col", String(length=10), nullable=nullable),
+                Column("string_default_col", String(), nullable=nullable),
+                Column("datetime_tz_col", DateTime(timezone=True), nullable=nullable),
+                Column("datetime_ntz_col", DateTime(timezone=False), nullable=nullable),
+                Column("date_col", Date, nullable=nullable),
+                Column("time_col", Time, nullable=nullable),
+                Column("float_col", Float, nullable=nullable),
+                Column("json_col", JSONB, nullable=nullable),
+                Column("bool_col", Boolean, nullable=nullable),
```

Comment:
TODO: Uuid column removed here, we can put it back with a conditional on sqlalchemy 2.0